packages/amplify_authenticator/lib/amplify_authenticator.dart

```diff
@@ -84,20 +88,18 @@ class Authenticator extends StatefulWidget {
   ///     - define
   ///
   Authenticator(
-
       {required this.child,
       SignInForm? signInForm,
       SignUpForm? signUpForm,
-      ConfirmSignInForm? confirmSignInForm}) {
-    this.signInForm = signInForm ?? DefaultForms.signInForm();
-    this.signUpForm = signUpForm ?? DefaultForms.signUpForm();
+      ConfirmSignInForm? confirmSignInForm,
+      AuthStringResolver? resolver}) {
```

Comment:
Wondering, would it be better if we have a default English resolver? So in that way in the future, we may want to store default resolvers in different languages.

Reply:
I don't think that is totally necessary - the customer can handle that through Flutter's OOTB internationaliztion functionality.